### PR TITLE
Add organizer logo to event pages

### PR DESCRIPTION
**Pull request by Arashfa0301**
# Description

Added organiser logo to event pages for user eye satisfaction and increased organiser pride. I am open to improvement suggestions. Important to note that the email link is removed since I believe it has minimal importance being here and it could be easily found at group page.

# Result

**before** 
image

**after**
image
# Testing

- I have thoroughly tested my changes on mobile and window views

---

### CODE REVIEW

File: app/routes/events/components/EventDetail/index.tsx [hunk 1]
```diff
@@ -86,6 +88,26 @@ const InterestedButton = ({ isInterested }: InterestedButtonProps) => {
   );
 };
 
+const groupLogoDict: { [key: string]: string } = {
```
Comment (ivarnakken) [inline app/routes/events/components/EventDetail/index.tsx:91]: Preferably just use the group logo, especially since other groups can also create events as well

File: app/routes/events/components/EventDetail/index.tsx [hunk 2]
```diff
@@ -323,27 +345,40 @@ export default class EventDetail extends Component<Props, State> {
...
         key: 'Arrangør',
         value: (
-          <span>
+          <Link to={groupLink} className={styles.organizerLogo}>
             {groupLink ? (
-              <Link to={groupLink}>{event.responsibleGroup.name}</Link>
+              <Tooltip
+                key={event.responsibleGroup.id}
+                content={<div>{event.responsibleGroup.name}</div>}
+                onClick={() => console.log(event.responsibleGroup.name)}
```
Comment (ivarnakken) [inline app/routes/events/components/EventDetail/index.tsx:353]: 👀

**Review by ivarnakken — CHANGES_REQUESTED** [review]
I'm not sure if I like the addition of the author's profile picture, since I don't think it is really needed, and I don't like "their face" becoming the face of the event, if that makes any sense ..

I think making group logos more visible is a nice touch though! 💯

**Comment by danielyanghansen** [comment]
Design-wise, I like the concept, and it's nice to have a visual connection to the organizer.
However, I feel it's important to keep the email address there somehow, as this is important for the participants to know who to contact.
Maybe show the email on hover?

**Comment by ivarnakken** [comment]
> However, I feel it's important to keep the email address there somehow, as this is important for the participants to know who to contact.
Maybe show the email on hover?

I vouch for adding it on hover. You can find the email by clicking on the link and going to the group page. I think people will manage fine.

File: app/routes/events/components/EventDetail/index.tsx (hunk 1 above)
Comment (Arashfa0301) [reply app/routes/events/components/EventDetail/index.tsx:91]: The get logo API is not working at this moment. I'll make an issue and maybe fix it myself.

File: app/routes/events/components/EventDetail/index.tsx (hunk 2 above)
Comment (Arashfa0301) [reply app/routes/events/components/EventDetail/index.tsx]: I see I see

**Comment by ivarnakken** [comment]
@Arashfa0301 
Remember to rebase from `master` instead of merging `master` into your working branch.

File: app/routes/events/components/EventDetail/index.tsx [hunk 3]
```diff
@@ -326,27 +348,37 @@ export default class EventDetail extends Component<Props, State> {
...
             key: 'Forfatter',
             value: (
-              <Link to={`/users/${event.createdBy.username}`}>
-                {event.createdBy.fullName}
-              </Link>
+              <span className={styles.organizerLogo}>
+                <ProfilePicture
+                  user={event.createdBy}
+                  size={40}
+                  alt={event.createdBy.firstName}
```
Comment (erlingfn) [inline app/routes/events/components/EventDetail/index.tsx:376]: Why not use the full name? A lot of people in this organization has the same name :)

File: app/routes/events/components/EventDetail/index.tsx [hunk 4]
```diff
@@ -326,27 +348,37 @@ export default class EventDetail extends Component<Props, State> {
...
-              event.responsibleGroup.name
+          <Link to={groupLink} className={styles.organizerLogo}>
+            {groupLink && (
+              <Tooltip
+                key={event.responsibleGroup.id}
+                content={event.responsibleGroup.name}
+              >
+                <CircularPicture
+                  alt={event.responsibleGroup.name}
+                  src={groupLogoDict[event.responsibleGroup.name]}
```
Comment (erlingfn) [inline app/routes/events/components/EventDetail/index.tsx:359]: How does this work if a group not in the `groupLogoDict` creates an event?

**Review by erlingfn — CHANGES_REQUESTED** [review]
I like the addition, but I think it has to be implemented in a generic and extendible way if it should be done

File: app/routes/events/components/EventDetail/index.tsx (hunk 3 above)
Comment (Arashfa0301) [reply app/routes/events/components/EventDetail/index.tsx:376]: I chose to remove the arranger picture. So the name will be as I was before

**Comment by Arashfa0301** [comment]
> I'm not sure if I like the addition of the author's profile picture, since I don't think it is really needed, and I don't like "their face" becoming the face of the event, if that makes any sense ..
> 
> I think making group logos more visible is a nice touch though! 💯

I have now removed it.

File: app/routes/events/components/EventDetail/index.tsx [hunk 5]
```diff
@@ -326,18 +327,21 @@ export default class EventDetail extends Component<Props, State> {
...
+              </Tooltip>
             )}{' '}
-            {event.responsibleGroup.contactEmail && (
-              <a href={`mailto:${event.responsibleGroup.contactEmail}`}>
-                {event.responsibleGroup.contactEmail}
-              </a>
-            )}
-          </span>
+            {event.responsibleGroup.name}
+          </Link>
```
Comment (ivarnakken) [inline app/routes/events/components/EventDetail/index.tsx:344]: hmm is the tooltip really needed now that the name is next to the image? I suggest either removing it or putting the email address inside.

Comment (Arashfa0301) [reply app/routes/events/components/EventDetail/index.tsx:344]: I shall remove it. The email looks very weird inside it 👍

File: app/routes/events/components/EventDetail/index.tsx [hunk 6]
```diff
@@ -315,26 +317,24 @@ export default class EventDetail extends Component<Props, State> {
           }
         : null,
     ];
-
+    c;
```
Comment (eikhr) [inline app/routes/events/components/EventDetail/index.tsx:320]: c;

File: package.json [hunk 7]
```diff
@@ -34,7 +34,7 @@
     "@loadable/component": "^5.15.3",
     "@reduxjs/toolkit": "^1.9.2",
     "@sentry/browser": "^7.36.0",
-    "@sentry/integrations": "^7.34.0",
+    "@sentry/integrations": "^7.36.0",
```
Comment (eikhr) [inline package.json:37]: Is this relevant to this PR?

File: app/routes/users/components/UserSettingsOAuth2.tsx [hunk 8]
```diff
@@ -90,15 +91,20 @@ const UserSettingsOAuth2 = (props: Props) => {
...
-          <Icon name="trash-outline" size={20} className={styles.deleteIcon} />
-        </Tooltip>
+          <Tooltip content="Fjern" style={{ marginTop: '-7px' }}>
+            <Icon
+              name="trash-outline"
+              size={18}
+              className={styles.deleteIcon}
+            />
+          </Tooltip>
+        </ConfirmModalWithParent>
```
Comment (eikhr) [inline app/routes/users/components/UserSettingsOAuth2.tsx:107]: This seems like a nice change, but it's not really relevant to the PR? At least mention it in the description

**Review by eikhr — CHANGES_REQUESTED** [review]
Looks good! I like the change. You need to rebase this though, and there are a few changes that seem unrelated to the PR though, perhaps left in accidentally from a merge conflict?

**Comment by ivarnakken** [comment]
Closing this as the branch is stale and too far behind. A rewrite is easier.
